R/aes-evaluation.R

```diff
@@ -313,7 +313,7 @@ strip_stage <- function(expr) {
   } else if (is_call(uq_expr, "stage")) {
     # Prefer stat mapping if present, otherwise original mapping (fallback to
     # scale mapping) but there should always be two arguments to stage()
-    uq_expr$after_stat %||% uq_expr$start %||% (if (is.null(uq_expr$after_scale)) uq_expr[[3]]) %||% uq_expr[[2]]
+    uq_expr$after_stat %||% uq_expr$start %||% uq_expr[[1]] %||% (if (is.null(uq_expr$after_scale)) uq_expr[[3]]) %||% uq_expr[[2]]
```

Comment:
I'm a bit unsure about the assumption that `uq_expr[[1]]` is the `start` argument.
Wouldn't it be safer to use `rlang::call_match(uq_expr, stage)` first?
I'm guessing that might also make the part after `if` redundant too.

Reply:
good call